vault/raft.go

```diff
@@ -324,6 +324,9 @@ func (c *Core) setupRaftActiveNode(ctx context.Context) error {
 		return err
 	}
 
+	// Run the verifier if we're configured to do so
+	raftBackend.RunRaftWalVerifier(c.activeContext)
+
```

Comment:
nit: (continued) yeah it seems the `start` prefix would be more consistent with the other calls here too.

Is there a reason to use `c.activeContext` vs the in-scope `ctx` like other calls?

Reply:
Nope, just an oversight on my part. I changed everything to use `Start` instead of `Run`.